torchvision/prototype/transforms/functional/_geometry.py

```diff
@@ -750,14 +750,12 @@ def pad_bounding_box(
 
     left, right, top, bottom = _parse_pad_padding(padding)
 
-    bounding_box = bounding_box.clone()
-
     # this works without conversion since padding only affects xy coordinates
```

Comment:
This should be moved inside the `else` branch, right? And should probably also be added to `crop_bounding_box` below.

Reply:
I'll remove this comment to avoid confusion

Reply:
Also fine by me.